### DISCUSSION TITLE
In ESP-01, how to assign Button option that can be used only to update WiFi credential.

### DISCUSSION THREAD

**Discussion by rarenu** [arendst/Tasmota#12214] [Q&A] [answered]
I am a newbie to Tasmota, some one please suggest me how to use the button option in ESP-01 that can only update WiFi credentials.

**Comment by barbudor** [comment]
Hi
I'm not sure to understand what you are looking for.
Can you please explain a little bit more ?

**Comment by rarenu** [comment, marked as answer]
Thank you. My need is, If I send my homemade 4 channel relay units to my
daughter. She should use it in her country just by Hardware resetting and
entering WiFi credentials of her home network using her laptop. I don't
want her to configure all the other options which I already done when I
tested it.

**Reply by sfromis** [reply]
You need no button for that, just remove your wifi ssid/password. Then she can connect to AP mode with her details.
`Backlog SSID1 0;Password1 0`

**Comment by rarenu** [comment]
Ok, thank you. Do I have to remove my ssid/password for both WiFiconfig 1
and 2 or 2 alone. Please clarify.

**Reply by sfromis** [reply]
I corrected my post. You do not need to do anything with `WifiConfig`. If you have two APs configured, better remove both.

**Comment by rarenu** [comment]
Thank you very much.
